Week7/Solution/solution6.py

```diff
@@ -0,0 +1,11 @@
+from pandas import *
+
+Column=['Product No', 'Product Name', 'January', 'February', 'March', 'April', 'May', 'June']
+dtf = read_csv('Student_Data.csv')
+
+dtf.loc[2.5] = [116, 'Portable Hardisk',90,78,76,45,34,22]
+dtf = dtf.sort_index().reset_index(drop=True)

```

Comment:
What is the use of reset_index()?

Reply:
if a new row is inserted in between the dataframe the index will be remain same so for the we need to write reset_index to reset the index